support-frontend/app/controllers/StripeController.scala

```diff
@@ -45,41 +45,47 @@ class StripeController(
   def createSetupIntentRecaptcha: Action[SetupIntentRequestRecaptcha] = PrivateAction.async(circe.json[SetupIntentRequestRecaptcha]) { implicit request =>
     val v2RecaptchaToken = request.body.token
 
-    val cloudwatchEvent = createSetupIntentRequest(stage, "v2Recaptcha");
+    val cloudwatchEvent = createSetupIntentRequest(stage, "v2Recaptcha")
     AwsCloudWatchMetricPut(client)(cloudwatchEvent)
 
-    if (v2RecaptchaToken.nonEmpty) {
-      val result = for {
-        recaptchaResponse <- recaptchaService.verify(v2RecaptchaToken, v2RecaptchaKey)
-        response <- if (recaptchaResponse.success) {
-          stripeService(request.body.stripePublicKey).map(response => Ok(response.asJson))
-        } else {
-          logger.warn(s"Returning status Forbidden for Create Stripe Intent Recaptcha request because Recaptcha verification failed")
-          EitherT.rightT[Future, String](Forbidden(""))
-        }
-      } yield response
-
-      result.fold(
-        error => {
-          logger.error(s"Returning status InternalServerError for Create Stripe Intent Recaptcha request because: $error")
-          InternalServerError("")
-        },
-        identity
-      )
-    } else {
-      logger.warn(s"Returning status BadRequest for Create Stripe Intent Recaptcha request because user provided no one-time-token value")
-      Future.successful(BadRequest("reCAPTCHA one-time-token required"))
-    }
+    val testPublicKeys = Set(testStripeConfig.australiaAccount.publicKey,
+      testStripeConfig.defaultAccount.publicKey,
+      testStripeConfig.unitedStatesAccount.publicKey)
```

Comment:
There's no recurring United States Stripe account, so this set may end up having an empty string, so maybe you should have a `filter(_.nonEmpty)`?

Reply:
Is that how it's currently configured? With an empty string?

Reply:
maybe we should remove `testStripeConfig.unitedStatesAccount.publicKey` from the set? 
If we aren't going to receive it from frontend then we shouldn't be checking against it in the backend?

There's no key in ssm - if it doesn't exist I think removing it from the code would be the clearer thing to do

Reply:
https://github.com/guardian/support-frontend/pull/2450/commits/9e5d6b4c201a1e459be8d485aa78c609f5189e26